### DISCUSSION TITLE
Where to file issues against linux-fslc?

### DISCUSSION THREAD

**Discussion by jaskij** [Freescale/meta-freescale#1404] [Q&A] [answered]
I'm not sure where to go - [linux-fslc](https://github.com/Freescale/linux-fslc/) has issues and discussions disabled.

I have found an issue specific to `linux-fslc` (not present in same version mainline) and have no idea where to send information about it.

---

Noting the info down so it doesn't get lost:

`add_one_port()` is called twice in `lpuart_probe()`, here:

https://github.com/Freescale/linux-fslc/blob/3ff5eb3ff57e665c47c072284d7f624e5452b85d/drivers/tty/serial/fsl_lpuart.c#L2965-L2986

Removing the second call seems to fix the issue (system has booted with console past the point, but I have not done extensive testing).

```patch
diff --git a/drivers/tty/serial/fsl_lpuart.c b/drivers/tty/serial/fsl_lpuart.c
index a12ff4954793..75e906c467d8 100644
--- a/drivers/tty/serial/fsl_lpuart.c
+++ b/drivers/tty/serial/fsl_lpuart.c
@@ -2981,10 +2981,6 @@ static int lpuart_probe(struct platform_device *pdev)
 	    sport->port.rs485.delay_rts_after_send)
 		dev_err(&pdev->dev, "driver doesn't support RTS delays\n");
 
-	ret = uart_add_one_port(&lpuart_reg, &sport->port);
-	if (ret)
-		goto failed_attach_port;
-
 	ret = devm_request_irq(&pdev->dev, sport->port.irq, handler, 0,
 				DRIVER_NAME, sport);
 	if (ret)
```

**Comment by MrCry0** [comment]
Thanks, @jaskij I'm fixing it.

**Comment by angolini** [comment, marked as answer]
https://github.com/Freescale/meta-freescale/issues/1406
https://github.com/Freescale/linux-fslc/pull/621
https://github.com/Freescale/meta-freescale/pull/1408

I think this is done, so I'm closing this thread
